lib/matplotlib/tests/test_colors.py

```diff
@@ -208,7 +208,9 @@ def test_Normalize():
     # and for scalar ones.
     eps = np.finfo(np.longdouble).resolution
     norm = plt.Normalize(1, 1 + 100 * eps)
-    assert_equal(norm(1 + 50 * eps), .5)
+    # This returns exactly 0.5 when longdouble is extended precision (80-bit),
+    # but only a value close to it when it is quadruple precision (128-bit).
+    assert 0 < norm(1 + 50 * eps) < 1
```

Comment:
You could also use one of the "almost equal" asserts from numpy (or nose, or pytest)